src/API/api_connect.js

```diff
@@ -0,0 +1,41 @@
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/revy0Qm0vprS3SsZDYIR/books';
+
+export const StoreApiSend = async (title, author, id) => {
+  const res = await fetch(BASE_URL, {
+    method: 'POST',
+    body: JSON.stringify(
+      {
+        item_id: id,
+        title,
+        category: author,
+      },
+    ),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  console.log('sended to api');
```

Comment:
**Optional Changes**
The previous Code Reviewer missed it, however;
- [ ] You shouldn't commit `console.log` to production. It makes your code look messy (code smell). 
Let us try to remove all the occurrences of `console.log` before merging.